src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,24 @@
+@import "../../patternfly-utilities";
+
 .pf-l-gallery {
-  display: grid;
-  grid-gap: 1rem; //revisit this value
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
-  padding: 0;
-  margin: 0;
+  $pf-l-gallery--GridGap: 1rem !default;
+  $pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(250px, 1fr)) !default;
+  $pf-l-gallery--Padding: 0 !default;
```

Comment:
I'd not make padding and margin a variable

Reply:
As a general rule I like setting the padding and margin explicitly because we don't always know what elements the class will be applied to. I can see scenarios (especially with layouts) where the elements will be using a <ul> or something similar, so in the spirit of modularity I set padding and margin.

Reply:
so the question remains, why would we make them a variable? I think they should be fixed values

Reply:
I misunderstood what you were asking. Yeah, we could definitely make them static.